library/sensu_metrics_check.py

```diff
@@ -50,6 +59,8 @@ def main():
                 command = '%s %s' % (module.params['prefix'], command)
             if module.params['use_sudo']:
                 command = "sudo %s" % (command)
+            if module.params['only_on_ip'] is not None and validIP(module.params['only_on_ip']):
+                command = 'if ip a | grep ' +  module.params['only_on_ip'] + ' > /dev/null 2>&1; then ' + command  + '; fi'
```

Comment:
if ip not exist,  ```command = "has ip %s && command" % (module.params['only_on_ip'])``` exit code will be not 0; and that will be warning or critical alert.